ghost/signup-form/.vscode/settings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "workbench.colorCustomizations": {
+    "editorIndentGuide.background": "#00000000",
+    "[Material Theme Palenight High Contrast]": {
+      "tab.activeBorder": "#00000000",
+      "tab.inactiveBackground": "#1b1e2b",
+      "activityBar.activeBorder": "#a6accd",
+      "editorGroupHeader.tabsBackground": "#1b1e2b",
+      "tab.border": "#00000000",
+      "sideBarSectionHeader.border": "#00000000",
+      "terminal.border": "#a6accd"
+    },
+    "activityBar.background": "#540E48",
+    "titleBar.activeBackground": "#761365",
+    "titleBar.activeForeground": "#FEFCFE"
+  }
+}
```

Comment:
Should probably be added to gitignore

Reply:
Oops, that was just from me opening the repo outside a workspace one time. I'll just delete it, shouldn't be added normally